setup.py

```diff
@@ -25,7 +25,8 @@
 # Dependencies
 extras_require = {
     'serial':   ['pyserial~=3.0'],
-    'neovi':    ['python-ics>=2.12']
+    'neovi':    ['python-ics>=2.12'],
+    'crccheck': ['crccheck>=0.6']
```

Comment:
The name should only specify the functionality that the user wants, e.g. the interface you want to use. The packages needed for that to work is not the user’s concern. In the future you may change what crc library to use.

In this case the name should be something like you had before and both pyserial and crccheck are needed.